examples/counter/store/configureStore.js

```diff
@@ -1,11 +1,21 @@
-import { createStore, applyMiddleware } from 'redux';
+import { createStore, applyMiddleware, compose } from 'redux';
 import thunk from 'redux-thunk';
-import rootReducer from '../reducers';
+import reducer from '../reducers';
+import switchReducer from './switchReducer';
 
-const createStoreWithMiddleware = applyMiddleware(
-  thunk
-)(createStore);
+function onHotReload(useReducer) {
```

Comment:
Could the param for `onHotReload()` be the path to the reducer module? Or does that not work because the require statement can't be inside `node_modules`?